frontend/js/src/explore/artist-similarity/artist-search/SearchItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ArtistType } from "./artistLookup";
```

Comment:
I see multiple places where types are imported from component files. I think a better solution would be to create a new `types.d.ts` file inside `explore/artist-similarity` folder and put all the types there except the interfaces for props.

Reply:
@arsh331, this feedback is still pending.